provider/cpu_utils.py

```diff
@@ -130,3 +130,22 @@ def check_cpu_flags(params, flags, test, session=None):
         logging.info("Check cpu flags on host")
         if missing:
             test.cancel("This host doesn't support flag %s" % missing)
+
+
+def cpu_check_cmd_output(check_cmd, check_output, session=None):
```

Comment:
This function seems to be a common one, not cpu specific.

Reply:
And it can be implemented by just running "check_cmd | grep check_output" inside guest. Not sure if we need a function.

Reply:
> And it can be implemented by just running "check_cmd | grep check_output" inside guest. Not sure if we need a function.

delete it, thanks.